### DISCUSSION TITLE
Где найти инструменты "Лайкер" и "Экспортер"?

### DISCUSSION THREAD

**Discussion by dparalon** [Chimildic/YaMuTools#25] [Помощь с ошибкой] [answered]
Добрый день! Для плейлистов отсутствуют инструменты "Лайкер" и "Экспортер", проверил на Google Chrome ver. 94.0.4606.81 (Official Build) (arm64) и на Firefox 93.0

**Comment by Chimildic** [comment, marked as answer]
Оба инструмента включаются в настройках. С расширения из магазинов, оба работают правильно. Проверил только что.

**Reply by dparalon** [reply]
Уточните, пожалуйста, где мне нужно выбрать "Экспортер"? Может я не вижу этой опции...
![image](https://user-images.githubusercontent.com/66110875/136530062-c8980002-f8e9-45b5-a980-d9acf225c0d7.png)
![image](https://user-images.githubusercontent.com/66110875/136530159-350b3bb3-87d4-4217-998a-3786e21ef34f.png)

**Reply by Chimildic** [reply]
Войдите на страницу плейлиста (не в боковом меню)

**Reply by dparalon** [reply]
спасибо! Всё прекрасно работает
